desktop/release-notes.md

```diff
@@ -24,13 +24,64 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.2
+
+{% include release-date.html date="2023-01-19" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
```

Comment:
Do you have the new checksums for the new builds yet @mat007 ?

Reply:
I have now, I’ll update that file.

Reply:
Done!